lib/index.js

```diff
@@ -54,17 +55,17 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         account, email, hash, clientId, serviceId,
         authenticationMethod, requiredAuthenticationMethods
       } = req.body;
-
       if(type === 'nonce') {
-        // permitted without authentication
+        // permitted without authentication. Do not require an account since
+        // anyone can request for a nonce.
         const {actor = null} = (req.user || {});
         await brAuthnToken.set({
           actor, account, email, type, clientId,
           authenticationMethod, requiredAuthenticationMethods
         });
         res.status(204).end();
       } else {
-        // all other types require authentication to create
+        // all other types require an authenticated account to create.
```

Comment:
I wouldn't change this, because `authentication` doesn't necessarily mean you have an account, it just means you are a known user of the system. There are bedrock systems where a user might actually not have "an account" but they are dynamically generated a `user`/`actor` on the system after authenticating.

```suggestion
        // all other types require authentication to create
```

Reply:
I think incorporating some of what longley has just said into the comment will prevent confusion in the future.